x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -463,6 +487,19 @@ const ActionsConnectorsList: React.FunctionComponent = () => {
           'xpack.triggersActionsUI.sections.actionsConnectorsList.multipleTitle',
           { defaultMessage: 'connectors' }
         )}
+        showWarningText={showWarningText}
+        warningText={i18n.translate(
+          'xpack.triggersActionsUI.sections.actionsConnectorsList.warningText',
+          {
+            defaultMessage:
+              '{connectors, plural, one {{singleWarningText}} other {{multipleWarningText}}} connector is used in a rule.',
```

Comment:
I think we can just do this:
`'{connectors, plural, one {This} other {At least one}} connector is used in a rule.',`. Is there a reason to define the warning texts in their own i18n?

Reply:
Oh cool! I will update this! No there is not a particular reason, I think I was just modeling it after some of the other text

Reply:
Resolved in this [commit](https://github.com/elastic/kibana/pull/137181/commits/dac37a909ba1e0cb375cd2635557acd48219c868)